frontend/src/screens/learner.tsx

```diff
@@ -184,28 +218,36 @@ export const DesktopStudyCards: FC<{studies: ParticipantStudy[]}> = ({ studies }
                     paddingBottom: '2rem',
                     paddingRight: '2rem',
                 }}
-                navigation={{
-                    enabled: true,
-                }}
-                modules={[FreeMode, Pagination, Navigation]}
+                modules={[FreeMode]}
             >
-                {studies.map(study => (
-                    <SwiperSlide style={{ padding: '1rem' }} key={study.id}>
-                        <StudyCard study={study} />
-                    </SwiperSlide>
-                ))}
+                {studies
+                    .sort((a, b) =>
```

Comment:
Maybe this can be extracted out to a function with a good name? its a little hard to read